### ISSUE TITLE
X axis data are over laps on each other

### ISSUE THREAD

**Issue by ravi-kachhadiya** [apexcharts/react-apexcharts#306]
I am getting the issue of overlapping x-axis data on each other when we zoom in or zoom out and the same issue occurs in large-size screen devices. so, there is any solution for this, I can control this?

![Screenshot (357)](https://user-images.githubusercontent.com/83212598/120309847-555e0a80-c2f3-11eb-84bd-b576e0389029.png)

**Comment by fluorspar20** [comment]
You can lower the tickAmount property of xaxis, if it doesn't affect your use-case.

**Comment by ravi-kachhadiya** [comment]
Yes is is affect my use-case. There is any other solution for that?

**Comment by ravi-kachhadiya** [comment]
> You can lower the tickAmount property of xaxis, if it doesn't affect your use-case.

Yes is is affect my use-case. There is any other solution for that?

**Comment by ashishEinfochips** [comment]
![Screenshot 2024-07-30 134917](https://github.com/user-attachments/assets/be1f785c-664a-4296-b717-cf85a7b2fc7b)
